File: backend/src/handlers/health.py
"""GET /healthz — cheap liveness check."""
import json
import os


def handler(event, context):
    return {
        "statusCode": 200,
        "headers": {
            "Content-Type": "application/json",
            "Access-Control-Allow-Origin": "*",
        },
        "body": json.dumps({
            "status": "ok",
            "service": "groundtruth-api",
            "version": os.getenv("APP_VERSION", "1.0.0"),
        }),
    }

File: backend/src/handlers/health.js
// GET /healthz — cheap liveness check for the deploy target and demo prep.
exports.handler = async () => {
  return {
    statusCode: 200,
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({ status: "ok", service: "groundtruth-api" }),
  };
};
